violet-jacobs-bio.html

```diff
@@ -0,0 +1,149 @@
+                Portfolio
+              </a>
+              <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownPortfolio">
+                <a class="dropdown-item" href="portfolio-1-col.html">1 Column Portfolio</a>
+                <a class="dropdown-item" href="portfolio-2-col.html">2 Column Portfolio</a>
+                <a class="dropdown-item" href="portfolio-3-col.html">3 Column Portfolio</a>
+                <a class="dropdown-item" href="portfolio-4-col.html">4 Column Portfolio</a>
+                <a class="dropdown-item" href="portfolio-item.html">Single Portfolio Item</a>
+              </div>
+            </li>
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownBlog" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Blog
+              </a>
+              <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownBlog">
+                <a class="dropdown-item" href="blog-home-1.html">Blog Home 1</a>
+                <a class="dropdown-item" href="blog-home-2.html">Blog Home 2</a>
+                <a class="dropdown-item" href="blog-post.html">Blog Post</a>
+              </div>
+            </li>
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownBlog" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Other Pages
+              </a>
+              <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownBlog">
+                <a class="dropdown-item" href="full-width.html">Full Width Page</a>
+                <a class="dropdown-item" href="sidebar.html">Sidebar Page</a>
+                <a class="dropdown-item" href="faq.html">FAQ</a>
+                <a class="dropdown-item" href="404.html">404</a>
+                <a class="dropdown-item" href="pricing.html">Pricing Table</a>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <div class="container">
+        <br>
+    <div class="row">
+        <div class="col-lg-6">
+            <h2>Violet Jacobs</h2>
+            <p>Violet Augusta Mary Frederica Kennedy-Erskine was born at the House of Dun near Montrose, on 1 September 1863, eldest daughter of William Henry Kennedy-Erskine, 18th laird of Dun, and his wife Catherine Jones, a Welsh landowner’s daughter. She was educated at home, but the House of Dun’s large library was open to her, and she showed an early interest in art and literature. In later writings she hinted at the ‘suffocating respectability’ of the life she and her siblings were meant to lead, though since it has been quoted about her (in Helen Cruickshank’s Octobiography) that she was ‘aye in and oot amo’ the ploomen’s feet at the Mains o’ Dun’, the children cannot have been entirely remote from ordinary life.</p>
```

Comment:
Everything here is all good. Though I do have a few suggestions, just so there's not a giant wall of text on the bios. Either by trimming down the text a little bit and have an extended version in a drop down should they wish to read more or have separate sections for the text in case they are looking for specific things.